content/W2016/W16-event-stephanie-talk.md

```diff
@@ -0,0 +1,36 @@
+Title: Circuit Breaker: How my Love for Writing Brought me to Tech
+Date: 2016-02-15
+Category: Events
+Tags: feminism, talks
+Slug: stephanie-morillo-talk
+Author: Fatema Boxwala
+Summary: Come out to hear Stephanie Morillo talk about her experiences in the
+tech world!
+
+WiCS Undergrad is proud to invite Stephanie Morillo as our out-of-town speaker
+this term! Stephanie will be joining us on March 21st, to talk about her 
+journey into tech.
+
+Abstract: The road to a successful career in tech wasn’t forged back in university; it was i
+forged in my mid-twenties when I was unemployed and learned to program on a 
+friend’s couch. After spending my first postgraduate years as a communications
+professional, I discovered the power of programming and with it, the exciting 
+and often frustrating realities of working in tech: the ability to interact 
+with people in the greater tech community, the chance to get exposed to new
+technologies, and the dearth of talent from underrepresented groups. In my 
+talk, I’ll recount my journey into tech, the observations I’ve made, and the 
+lessons I’ve learned that I wish I could tell my university-aged self about 
+what it means to be successful, and how to dismantle limiting beliefs before 
+finding one’s first job.
+
+**About Stephanie**: Stephanie is a writer, technologist, and musician based in
+the Bronx, who frequently writes and gives talks about class, race and the
+tech industry. Check out her [website](http://stephaniemorillo.com) and 
+[twitter](https://twitter.com/radiomorillo?lang=en)!
+
+## Event Details ##
+
++ **Who:** Everyone!
```

Comment:
I believe Stephanie Morillo is the who here, unless you want to put "open to the public"